### DISCUSSION TITLE
Could Not Find Repository

### DISCUSSION THREAD

**Discussion by aoniani** [community/community#88706] [Actions] [answered]
### Select Topic Area

Question

### Body

it can't find repository, so i can't push or pull any commits.
link is working perrfectly 1 thing i noticed is that in my .git\refs\remotes\origin file, Masted file is missing
i copied master file from heads folder but it still wont work

**Comment by Shahnawazk7944** [comment]
hey @aoniani

Try fetching recent changes from the remote repository using `git fetch origin`. This might automatically recreate the missing "Master" file. If it works, then no need to worry. Try pushing/pulling again. If it does not work, run `git remote -v` to ensure the remote URL is correct and points to the intended repository. If the URL is wrong, update it using `git remote set-url origin <correct_url>`. If it still does not work, then the last thing we can do to track branches is to run `git branch -u origin/master master` to explicitly set your local "master" branch to track the remote "master" branch.

**Reply by aoniani** [reply]
i found what was the problem, i was using 2 github accounts. i was added to project with only one of them . so even when i was logged in with account that i was added to project, it still uses other account for commits.Do you know how to remove that other account? or is there any way i can  make it work because i have access to github copilot on other account and connected to it because of it.

**Reply by Shahnawazk7944** [reply, marked as answer]
Hi @aoniani 

I think you are globally logged in with your previous account if you want to check of which account with you are logged in open a terminal and run `git config --global user.name` and `git config --global user.email`. These commands show your globally configured Git username and email.
Ensure the username and email displayed match the your present account you want to use for commits. If they don't, run `git config --global user.name "<your desired username>"` and `git config --global user.email "<your desired email>"` to update them with your latest one which you want to use (your GitHub copilot account).
